lib/db.js

```diff
@@ -64,6 +65,14 @@ module.exports.getUnexpiredConference = async (id) => {
         const conferences = await knex("conferences").select()
             .where({ id: id })
             .andWhere("expiresAt", ">", new Date())
+        // Format the conferenceDays to be YYYY-MM-DD strings, rather than full Date objects
+        // (the field is a table.date in knex, not a table.datetime, so it's less confusing to use a dateString.)
+        conferences.forEach(conference => {
+          if (conference.conferenceDay) {
+              const dayString = format.formatStandardDate(conference.conferenceDay)
+              conference.conferenceDay = dayString
```

Comment:
l'économie d'une ligne ?

```suggestion
              conference.conferenceDay = format.formatStandardDate(conference.conferenceDay)
```